packages/rancher-monitoring/templates/crd-template/values.yaml

```diff
@@ -7,5 +7,6 @@ global:
     systemDefaultRegistry: ""
```

Comment:
I don't think this crd-template/values.yaml would get picked up by the images.txt script. I meant to add it here, this will get picked up: https://github.com/rancher/charts/blob/dev-v2.6/charts/rancher-monitoring/rancher-monitoring/100.0.0%2Bup16.6.0/values.yaml

Add comment something like:
> This is used in the CRD chart, keep this inline with that, and the rancher shell setting to avoid missing tag in rancher-images.txt